tests/Unit/Executables/CMakeLists.txt

```diff
@@ -30,3 +31,35 @@ add_dependencies(
   test-executables
   ConvertComposeTable
   )
+
+add_test(
+  NAME "Unit.Executables.ExtendConnectivity"
+  COMMAND
+  ${SHELL_EXECUTABLE}
+  -c
+  "rm -f ${_CONNECTIVITY_OUTPUT} && \
+   cp ${CMAKE_SOURCE_DIR}/tests/Unit/IO/Test_ExtendConnectivity_Data0.h5 \
+   ${CMAKE_BINARY_DIR}/tmp && \
+   ${CMAKE_BINARY_DIR}/bin/ExtendConnectivity \
+   --file_name ${CMAKE_BINARY_DIR}/tmp/Test_ExtendConnectivity_Data0.h5 \
+   --subfile_name VolumeData && \
+   ${CMAKE_BINARY_DIR}/bin/spectre generate-xdmf \
+   --output ${_CONNECTIVITY_OUTPUT} \
+   --subfile-name VolumeData \
+   ${CMAKE_BINARY_DIR}/tmp/Test_ExtendConnectivity_Data0.h5 \
+   && rm ${CMAKE_BINARY_DIR}/tmp/Test_ExtendConnectivity_Data0.h5 \
+   && rm ${_CONNECTIVITY_OUTPUT}.xmf"
+  )
+set_standalone_test_properties("Unit.Executables.ExtendConnectivity")
+set_tests_properties(
+    "Unit.Executables.ExtendConnectivity"
+    PROPERTIES
+    LABELS "standalone;unit")
```

Comment:
"standalone;unit" doesn't make sense.  Those are mutually exclusive types, along with "inputfiles" (despite us naming everything with a "Unit." prefix).  Drop this `set_test_properties` and the following `add_dependencies` for `unit-tests`.